.github/workflows/android_emerge_upload.yml

```diff
@@ -25,7 +25,8 @@ jobs:
       - name: Setup Gradle build action
         uses: gradle/actions/setup-gradle@v4
       - name: Emerge analysis
-        run: ./gradlew :app:emergeUploadReleaseAab
+        run: ./gradlew :app:bundleRelease :app:emergeUploadReleaseAab
```

Comment:
The emerge plugin doesn't call this task even though it obtains the result so therefore we need to explicitly call `bundleRelease` because the sentry plugin adds a dependency from this task.